lessons/codap-transformers2/langs/en-us/pages/notes-sort-transformer.adoc

```diff
@@ -0,0 +1,14 @@
+= Using Transformers: Sort, Filter, Transform Attribute, Build Attribute
+
+*Transformers* allow us to transform datasets to produce new, distinct output datasets, instead of modifying the original input dataset itself. We use them to manipulate tables and enable low-stakes "what if?" exploration.
+
+- In this course, we'll be using a variety of Transformers.
+
+** `Sort` - We must provide `Sort` with a dataset, a sort method (we'll focus on sorting by attribute in this course), and a sort direction. `Sort` produces a copy of the input dataset, with cases sorted by the selected attribute.
```

Comment:
I'd re-order these to use the same order in which they are introduced

Reply:
good call.